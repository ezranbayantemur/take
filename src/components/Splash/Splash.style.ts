import {color, font} from '@style';
import {StyleSheet} from 'react-native';

export default StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
  logo_text: {
    color: color.MONZA,
    fontFamily: font.BOLD_ITALIC,
    fontSize: 90,
  },
});
