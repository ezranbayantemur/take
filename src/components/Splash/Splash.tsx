import {SafeAreaView, Text} from 'react-native';
import React from 'react';
import styles from './Splash.style';

const Splash = () => {
  return (
    <SafeAreaView style={styles.container}>
      <Text style={styles.logo_text}>take</Text>
    </SafeAreaView>
  );
};

export default Splash;
