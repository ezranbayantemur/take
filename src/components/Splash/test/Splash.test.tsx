import React from 'react';
import {render} from '@testing-library/react-native';
import Splash from '../Splash';

describe('Splash tests', () => {
  let wrapper: ReturnType<typeof render>;

  beforeEach(() => {
    wrapper = render(<Splash />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
