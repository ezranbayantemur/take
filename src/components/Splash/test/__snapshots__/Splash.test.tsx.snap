// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash tests should render correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "#DB040E",
        "fontFamily": "Lato-BoldItalic",
        "fontSize": 90,
      }
    }
  >
    take!
  </Text>
</RCTSafeAreaView>
`;
